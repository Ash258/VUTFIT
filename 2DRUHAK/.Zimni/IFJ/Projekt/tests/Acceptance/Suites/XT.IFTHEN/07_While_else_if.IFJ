' While with elseif in condition

scope
	if 78 > 2 then
	    dim a as integer = 12
		do while a >= 1
			if a > 8 then
				print 69;
			elseif a > 3 then
				print 74;
			else
				print 11;
			end if
			a = a - 1
		loop
	else
		print 77;
	end if
end scope
