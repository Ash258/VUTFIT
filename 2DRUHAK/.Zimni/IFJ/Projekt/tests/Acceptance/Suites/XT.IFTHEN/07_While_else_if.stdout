 69 69 69 69 74 74 74 74 74 11 11 11